### PR TITLE
Document for deploying model in remote TorchServe

### PR DESCRIPTION
**Pull request by shrinath-suresh**
Signed-off-by: Shrinath Suresh <shrinath@ideas2it.com>

## What changes are proposed in this pull request?

Document for deploying model in remote TorchServe

## How is this patch tested?

NA

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow TorchServe Deployment Plugin users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?
Components
- [ ] `area/deploy`: Main deployment plugin logic
- [ ] `area/build`: Build and test infrastructure for MLflow TorchServe Deployment Plugin
- [ ] `area/docs`: MLflow TorchServe Deployment Plugin documentation pages
- [ ] `area/examples`: Example code

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [x] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by HamidShojanazeri — COMMENTED** [review]
@shrinath-suresh  Looks fine to me.

File: docs/remote-deployment.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+Setting up Config Properties:
+-----------------------------
+
+Set the management and inference URL in config properties file. `config.properties` file is placed in the home directory of the repository.
+
+For example:
+
+.. code-block::
+
+    inference_address=http://18.225.35.109:8080
```
Comment (chauhang) [inline docs/remote-deployment.rst:54]: @shrinath-suresh Mention the IP address here needs to be that of the remote torchserve

File: docs/remote-deployment.rst [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    http://eda154810618.ngrok.io/mnist_test.mar
+
+mnist_test.mar file should be downloaded.
+
+We are all set for performing registration. To register the model in remote torchserve instance run 
+
+.. code-block::
+
+    python register.py --mar_file_name mnist_test.mar
```
Comment (chauhang) [inline docs/remote-deployment.rst:147]: @shrinath-suresh Where is the register.py file located? Not seeing it in the MNIST example folder

**Review by chauhang — CHANGES_REQUESTED** [review]
Please see comments inline

File: docs/remote-deployment.rst (hunk 2 above)
Comment (shrinath-suresh) [reply docs/remote-deployment.rst:147]: @chauhang We no longer need a separate script for registering the model. As discussed, the model registration can be done by adding a parameter in log_model . I have updated the code to perform registration in `mnist_model.py` itself - https://github.com/mlflow/mlflow-torchserve/blob/70baee835723b0b61421ed37840a999bf7eb1ec6/examples/MNIST/mnist_model.py#L301

I will update the documentation.

File: docs/remote-deployment.rst (hunk 1 above)
Comment (shrinath-suresh) [reply docs/remote-deployment.rst:54]: Instead of IP address, i will add a tag <REMOTE_TORCHSERVE_IP>

File: docs/remote-deployment.rst (hunk 2 above)
Comment (shrinath-suresh) [reply docs/remote-deployment.rst]: I have updated the steps

File: docs/remote-deployment.rst [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+.. code-block:: 
+
+    curl "http://<Remote_TS_IP>:8081/models"
+
+This command should retrieve the list of existing models present in remote torchserve. This is an optional step. If you are sure about the connectivity proceed with the deployment steps as stated below.. 
+
+Training the model:
+-------------------
+
+Clone the code from - `https://github.com/mlflow/mlflow-torchserve/tree/remote-torchserve-fix <https://github.com/mlflow/mlflow-torchserve/tree/remote-torchserve-fix>`_ and move to `mlflow-torchserve/examples/MNIST` folder.
```
[the remark this answers is not in the document]
Comment (shrinath-suresh) [reply docs/remote-deployment.rst]: I have updated the URL

File: docs/remote-deployment.rst (hunk 2 above)
Comment (shrinath-suresh) [reply docs/remote-deployment.rst]: I got confused with model registration and torchserve mar file registration and deleted the file. I have added it back - https://github.com/mlflow/mlflow-torchserve/pull/47/files#diff-a408565ad545aced7e32600c579119784fa4b1d88867d62205f44e1806a052a4

**Review by chauhang — APPROVED** [review]

**Pull request merged**